tools/train.py

```diff
@@ -251,6 +269,29 @@ def main():
                 for loss_name, loss_value in stats.items():
                     vdl_writer.add_scalar(loss_name, loss_value, vdl_loss_step)
                 vdl_loss_step += 1
+        
+        """
+        if it == 200:
+            profiler.start_profiler("All")
+            #fluid.core.nvprof_start()
+        if it == 202:
+            #fluid.core.nvprof_stop()
+            profiler.stop_profiler("total", "./profile")
+            return
+        """
+        
+        
+        if it == 200:
```

Comment:
Because calculate cost time is at below, so it is not necessary,  can be delete.

Reply:
ok, it's used to show the speed. thanks.

Reply:
ok, it's used to show the speed. thanks.